packages/react/src/hooks/accounts/useConnect.ts

```diff
@@ -45,6 +45,7 @@ export function useConnect({
     onMutate,
     onSettled,
     onSuccess,
+    context: queryClientContext,
```

Comment:
Can we have our own `useMutation` implementation (under `utils/query`) so we aren't having to manually pass down `context` on every usage?